### PR TITLE
Add bugzilla information to OWNERS

### PR DESCRIPTION
**Pull request by beekhof**
See thread  'Establishing a link between BZ components and OpenShift artifacts'

```release-note
NONE
```

### CODE REVIEW

File: OWNERS [hunk 1]
```diff
@@ -7,3 +7,12 @@ filters:
     approvers:
       - yanirq
       - ksimon1
+
```
Comment (yanirq) [inline OWNERS:10]: @beekhof  I would add you here to approvers list as well

Comment (beekhof) [reply OWNERS:10]:
```suggestion
      - beekhof
```

**Comment by yanirq** [comment]
/approve
/lgtm
/hold 
We can remove hold once we see why standrad-ci is failing (although this change should affect it i presume)

File: OWNERS [hunk 2]
```diff
@@ -7,3 +7,12 @@ filters:
     approvers:
       - yanirq
       - ksimon1
+      - beekhof
```
Comment (beekhof) [inline OWNERS:10]:
```suggestion
      - beekhof

```

**Comment by beekhof** [comment]
restoring /lgtm from @yanirq

**Pull request merged**